packages/mobile/src/transactions/feed/query.ts

```diff
@@ -0,0 +1,88 @@
+export const TRANSACTIONS_QUERY = `
```

Comment:
I tried, but it doesn't work with it. I get this error:
`GraphQL queries must be strings. It looks like you're sending the internal graphql-js representation of a parsed query in your request instead of a request in the GraphQL query language. You can convert an AST to a string using the `print` function from `graphql`, or use a client like `apollo-client` which converts the internal representation to a string for you`

Reply:
Ah yes I see, we're not using Apollo client with this one. Nevermind.